assets/src/admin/paired-browsing/client.js

```diff
@@ -1,35 +1,91 @@
 /**
  * WordPress dependencies
  */
-import domReady from '@wordpress/dom-ready';
+// import domReady from '@wordpress/dom-ready';
+
+function domReady( callback ) {
+	if (
+		document.readyState === 'complete' || // DOMContentLoaded + Images/Styles/etc loaded, so we call directly.
+		document.readyState === 'interactive' // DOMContentLoaded fires at this point, so we call directly.
+	) {
+		return callback();
+	} // DOMContentLoaded has not fired yet, delay callback until then.
+
+	document.addEventListener( 'DOMContentLoaded', callback );
+	return null;
+}
```

Comment:
I did this because I was getting mysterious sporadic failures with the imported `domReady`:

![image](https://user-images.githubusercontent.com/134745/69913877-2356b880-1470-11ea-9316-122d785e21ee.png)